packages/devtools_app/lib/src/analytics_notification.dart

```diff
@@ -0,0 +1,196 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
+
+import 'theme.dart';
+import 'ui/analytics.dart' as analytics;
+import 'ui/analytics_platform.dart' as platform;
```

Comment:
analytics_platform has a direct dependency on dart:html, which will be problematic for the desktop version of DevTools. Have you tested the desktop version? I would expect something would be blowing up

Reply:
Moved things around. It's now using a conditional import and uses a stub on the desktop version.